src/community/index.html

```diff
@@ -6,20 +6,19 @@
 
 <section class="community__intro row text-center">
     <div class="col-12">
-        <h1>Welcome to the Flutter community</h1>
+        <h1>Chào mừng đến với cộng đồng Flutter</h1>
         <p class="community__intro__description">
-            We're open source and would love to hear from you!
-            Below you'll find ways to get involved in the Flutter
-            community as well as links to
-            resources that can help answer your questions.
-            If you're in China, read
-            <a href="/community/china">Using Flutter in China</a>.
+            Chúng tôi là dự án mã nguồn mở và rất muốn nghe ý kiến từ bạn!
+            Ở bên dưới là cách để tham gia vào cộng đồng Flutter cũng như
+            những đường dẫn đến những nguồn có thể giúp bạn giải đáp những thắc mắc.
+            Nếu bạn ở Trung Quốc, hãy đọc
+            <a href="/community/china">Sử dụng Flutter ở Trung Quốc</a>.
         </p>
     </div>
 </section>
 
 {% comment %}
-    Featured services
```

Comment:
Nit: this change is not needed, and might cause problem when merging w/ upstream later. But I'll let it pass for now.